contract/system/param.go

```diff
@@ -26,7 +70,10 @@ const (
 )
 
 var (
-	systemParams parameters
+	systemParams *parameters = &parameters{
+		mutex:  sync.RWMutex{},
+		params: map[string]*big.Int{},
+	}
```

Comment:
I suspect that the above can be just `systemParams *parameters`
It is loaded on `loadParams()`